src/components/KeyItem/styles.js

```diff
@@ -0,0 +1,101 @@
+import styled from 'styled-components'
+import { StyledLabel } from '@components/LocaleLabel/styles'
+import { StyledTag } from '@components/Tag/styles'
+
+export const KeyWrapper = styled.div``
+
+export const KeyDetails = styled.div``
+
+export const KeyValueColumn = styled.div``
+
+export const KeyValue = styled.div`
+  display: inline-block;
+  font-family: ${({ theme }) => theme.fonts.families.sourceCode};
+  font-size: ${({ theme }) => theme.fonts.sizes.default};
+  padding: 0.25em;
+  background: ${({ theme }) => theme.colors.lightGray};
+  border-radius: ${({ theme }) => theme.globals.radius};
+  user-select: text;
+`
+
+export const KeyTranslationColumn = styled.div`
+  display: inline-block;
+  align-self: center;
+
+  ${StyledLabel} {
+    margin-right: ${({ theme }) => theme.sizes.tiny};
+  }
```

Comment:
I see the benefits, but I feel like you’re using this pattern of component selectors a lot!
Here I would have probably done:
```js
export const KeyTranslationLabel = StyledLabel.extend`
  margin-right: ${({ theme }) => theme.sizes.tiny};
`
```

But maybe that’s just personal preference, unambiguity by creating another component VS simplicity. 😄 


From the docs, https://www.styled-components.com/docs/advanced#referring-to-other-components:
> This allows for extremely powerful composition patterns without having to fuss around with naming and avoiding selector collisions.

So I guess it’s a perfectly valid use-case… I’ll get used to it! 😄

Reply:
Yes, of course this is a valid use-case! It avoids the creation of a new component. 
This generates a child selector, like so:
`.parent-component .child-component { margin-right: 10px; }`

I tend to find this pattern way more convenient in this case.